v20.2/import-performance-best-practices.md

```diff
@@ -0,0 +1,129 @@
+
+{{site.data.alerts.callout_info}}
+If the import size is small (around 64-129 MiB), then you do not need to do anything to optimize performance. In this case, the import should run quickly, regardless of the settings.
+{{site.data.alerts.end}}
+
+## Split your data into multiple files
+
+Splitting the import data into multiple files can have a large impact on the import performance. We recommend splitting your data into as many files as there are nodes.
+
+For example, if you have a 3-node cluster, split your data into 3 files and import:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> IMPORT TABLE customers (
+        id UUID PRIMARY KEY,
+        name TEXT,
+        INDEX name_idx (name)
+)
+CSV DATA (
+    'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/customers.csv',
+    'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/customers_2.csv',
+    'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/customers_3.csv',
+);
+~~~
+
+CockroachDB imports the files that you give it, and does not further split them. For example, if you import one large file for all of your data, CockroachDB will process that file on one node– even if you have more nodes available. However, if you import two files (and your cluster has at least two nodes), each node will process a file in parallel. This is why splitting your data into as many files as you have nodes will dramatically decrease the time it takes to import data.
+
+{{site.data.alerts.callout_info}}
+If you split the data into _more_ files than you have nodes, it will not have a large impact on performance.
+{{site.data.alerts.end}}
+
+## Choose a performant import format
+
+Import formats do not have the same performance because of the way they are processed. Below, import formats are listed from fastest to slowest:
+
+1. [`CSV`](migrate-from-csv.html) or [`DELIMITED DATA`](import.html#delimited-data-files) (both have about the same import performance)
+1. [`AVRO`](migrate-from-avro.html)
+1. [`PGCOPY`](import.html)
+1. [`MYSQLDUMP`](migrate-from-mysql.html)
+1. [`PGDUMP`](migrate-from-postgres.html)
+
+We recommend formatting your import files as `CSV`, `DELIMITED DATA`, or `AVRO`. These formats can be processed in parallel by multiple threads, which increases performance.
+
+However, `PGCOPY`, `MYSQLDUMP`, and `PGDUMP` run a single thread to parse their data, and therefore perform substantially slower.
+
+`MYSQLDUMP` and `PGDUMP` are two examples of “bundled” data. This means that the dump file contains both the table schema as well as the data to import.
+
+These formats are the slowest to import, with `PGDUMP` being the slower of the two. This is because CockroachDB has to first load the whole file, read the whole file to get the schema, and then import the data. While these formats are slow, there are a couple of things you can do to speed up a bundled data import:
```

Comment:
`read the whole file to get the schema`, create the table with that schema, and then import data`? the value to me of adding the table creation comment is that, well, it could fail on table creation.